extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -83,12 +83,6 @@ export const SAFEFRAME_VERSION_HEADER = 'X-AmpSafeFrameVersion';
 /** @type {string} @visibleForTesting */
 export const EXPERIMENT_FEATURE_HEADER_NAME = 'amp-ff-exps';
```

Comment:
unrelated changes, let's revert

Reply:
will do - there are days I hate git and days I just loath it.